src/RFC-0131_Mining.md

```diff
@@ -135,6 +134,9 @@ The hash MUST be constructed with the following encoded with [consensus encoding
 - total kernel offset
 - total script offset
 
+This hash is used in both the SHA-3 and RandomX proof-of-work algorithms. The header version for the Tari Genesis 
+block is 1.
```

Comment:
nit: We currently count from 0, 
Either we need to update this to 0, or we need to update the code to count the header version from 1.

Reply:
The code already has the current version at 1.
Version 0 was for Dibbler.

Reply:
we had 0 for esme as well. 
https://github.com/tari-project/tari/blob/6ff1c02d3451d856a7c0c979109aaae99dc38ca1/base_layer/core/src/consensus/consensus_constants.rs#L519

Reply:
we changed to 1 when we changed the PoW

Reply:
Ok, but you agree that the current version is 1; and as the code stands, the genesis block will also be version 1.

Reply:
Going to close this convo with the following:
- Since it currently matches the code, it's better to leave as is. Otherwise auditors will flag it.
- If the code changes ahead of mainnet, this RFC must be amended to match.